### PR TITLE
[Mobile] - Update LaunchScreen logo and app icon

### PR DESCRIPTION
**Pull request by geriux**
## What?
This PR proposes updating the LaunchScreen logo and app icon for the demo iOS app.

## Why?
Currently, we have a text logo and the default LaunchScreen placeholders since the project was created. Even though this is the demo app, it's nice to customize it a bit.

## How?
It uses the Gutenberg logo and it's applied to the LaunchScreen and app icon.

## Testing Instructions
The app should launch as expected with the new icon and LaunchScreen logo.

### Testing Instructions for Keyboard
N/A

## Screenshots or screencast 
Before|After
-|-
<kbd>|<kbd></kbd>

App Icon|LaunchScreen
-|-
<kbd>|<kbd></kbd>

### CODE REVIEW

**Review by dcalhoun — APPROVED** [review]
Very cool! 🚀 :shipit: 

I might suggest the Gutenberg logo with all of its lines is too busy to use as the app icon. WDYT about changing that or both the icon and launch screen to use the wordmark only? I.e. just the G displayed on [gutenberg.run](http://gutenberg.run) or atop the [demo page](https://wordpress.org/gutenberg/). To be clear, this proposal should not block this PR. 

image

**Comment by geriux** [comment]
> I might suggest the Gutenberg logo with all of its lines is too busy to use as the app icon. WDYT about changing that or both the icon and launch screen to use the wordmark only? I.e. just the G displayed on [gutenberg.run](http://gutenberg.run) or atop the [demo page](https://wordpress.org/gutenberg/). To be clear, this proposal should not block this PR.

Ohhh nice suggestion! I like that better! I'll push the changes before merging 🚀  Thanks for the review and the suggestion 🙇

**Pull request merged**
